src/main/java/org/terasology/ligthandshadow/componentsystem/input/TabButton.java

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+package org.terasology.ligthandshadow.componentsystem.input;
+
+import org.terasology.engine.input.BindButtonEvent;
+import org.terasology.engine.input.DefaultBinding;
+import org.terasology.engine.input.RegisterBindButton;
+import org.terasology.input.InputType;
+import org.terasology.input.Keyboard;
+
+@RegisterBindButton(id = "statistics", description = "Show statistics screen", category = "general", repeating = true)
+@DefaultBinding(type = InputType.KEY, id = Keyboard.KeyId.V)
+public class TabButton extends BindButtonEvent {
```

Comment:
From the name it's not clear that this is an event.
Also I'm missing documentation.
And I wonder whether you mean ["tap"](https://www.oxfordlearnersdictionaries.com/definition/english/tap_1), not ["tab"](https://www.oxfordlearnersdictionaries.com/definition/english/tab_2)? The second explanation for "tab" might fit, but is ambiguous IMO as we're not actually using the tab key...

Reply:
Most of the button based events I have seen don't explicitly mention event in their name.

Reply:
@jdrueckert and I briefly discussed this offline. All the button events (for keys on the keyboard) don't have the suffix, so omitting it for consistency makes sense :+1: 

The only events where we use an `...Event` suffix is for button events associated with mouse clicks - for whatever reason :shrug: